### DISCUSSION TITLE
Emphasis and Additional Features

### DISCUSSION THREAD

**Discussion by SamBigAbs** [azuritecoin/OnnxDiffusersUI#23] [Ideas]
Hi. Your tutorial and GUI were helpful in getting started. I have a few questions that I was hoping you could answer or point me towards the files that may have the answer. 

Does your script recognize any form of emphasis? In Automaic1111 webgui, brackets ie "(tag:1.5)" modifies the weight of the prompt. Does your GUI script do this or does the diffuser handle that?

Would it be possible, and simple, to output the result every 5 or 10 steps? Would this be a helpful feature? I have seen examples where changing the steps alters the image significantly. If I do 100 steps it would be nice to get 9 images then choose my favourite.

**Comment by azuritecoin** [comment]
I haven't implemented prompt emphasis yet. I'm waiting for the next release of diffusers before adding more features. You could also modify the script yourself if you want to. See this: https://github.com/huggingface/diffusers/tree/main/examples/community#long-prompt-weighting-stable-diffusion

It's possible to have it output every few steps. But it does involve modifying the package files. I'll look into it but again I'm waiting for next release of diffusers.
